sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/CreateTableExec.scala

```diff
@@ -54,3 +55,36 @@ case class CreateTableExec(
 
   override def output: Seq[Attribute] = Seq.empty
 }
+
+case class CreateTableStagingExec(
```

Comment:
Why use a staged table for a simple create? If there isn't a functional difference then I'd rather not create two plans.